crypto/elliptic-curve-examples.ts

```diff
@@ -55,3 +60,21 @@ const CurveParams = {
   Pallas: pallasParams,
   Vesta: vestaParams,
 };
+
+// https://datatracker.ietf.org/doc/html/rfc8032#section-5.1
+const ed25519Params: TwistedCurveParams = {
+  name: 'Ed25519',
+  modulus: exampleFields.f25519.modulus, // 2^255 - 19
+  order: 0x1000000000000000000000000000000014def9dea2f79cd65812631a5cf5d3edn, //2^252 + 27742317777372353535851937790883648493,
+  cofactor: 8n,
```

Comment:
This is an optional parameter, please a pair of eyes to double check it's correct.

Reply:
LGTM:
```
|     c     | base 2 logarithm of cofactor of edwards25519 in       |
|           | [[RFC7748](https://datatracker.ietf.org/doc/html/rfc7748)] (i.e., 3)
   ```